runtime/oak/src/lib.rs

```diff
@@ -305,7 +305,7 @@ impl frame_system::Config for Runtime {
 
 parameter_types! {
 	// One storage item; key size is 32; value is size 4+4+16+32 bytes = 56 bytes.
-	pub const DepositBase: Balance = deposit(1, 88);
+	pub const DepositBase: Balance = deposit(1, 88); // IRSAL: but this says 88? Intentional?
```

Comment:
@rhuttman - any ideas on this? Charles implemented multi-sig, but i wanted to make sure this makes sense to you

Reply:
this was lifted straight from the polkadot runtime so should be fine